src/feature.rs

```diff
@@ -44,7 +44,7 @@ pub fn filterbanks(
     sampling_freq: f64,
     low_freq: Option<f64>,
     high_freq: Option<f64>,
-) -> ndarray::ArrayBase<ndarray::OwnedRepr<_>, ndarray::Dim<[usize; 2]>> {
+) -> ndarray::ArrayBase<ndarray::OwnedRepr<f32>, ndarray::Dim<[usize; 2]>> {
```

Comment:
For the time being, we should stick to f64, at least for anything that's not input. While f32 is faster, we don't know if the lower precision is going to affect the result by too much. Once we have a working implementation let's try f32.

Also, can be written as `Array2<f64>`